content/es/loosely-coupled-architecture.md

```diff
@@ -2,17 +2,16 @@
 title: Arquitectura débilmente acoplada
 status: Completed
 category: Propiedad
-tags: ["fundamento", "arquitectura", "propiedad"]
+tags: ["básico", "arquitectura", "propiedad"]
 ---
 
-
-La arquitectura débilmente acoplada es un tipo de arquitectura en donde los componentes individuales de una aplicación se construyen de manera independiente unos de los otros 
-(es el paradigma opuesto a la [arquitectura fuertemente acoplada](/es/tightly-coupled-architectures/)). 
-Cada componente, a veces denominado [microservicio](/es/microservices/), 
-está diseñado para realizar una función específica de manera que pueda ser utilizado por cualquier número de servicios. 
-Este patrón es generalmente más lento de implementar que la arquitectura estrechamente acoplada, 
+La arquitectura débilmente acoplada es un tipo de arquitectura en donde los componentes individuales de una aplicación se construyen de manera independiente unos de los otros
```

Comment:
Algunos saltos de línea fueron agregados a propósito para que coincidieran con la versión en inglés y facilitara la revisión. Quizá este caso es uno de ellos

Reply:
Is the markdown auto-formatter, will redó this commit.